FR #966: Add recency to metadata (#1374)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: specification/appendix/metadata_examples/recency_metadata_example.md

# Recency Metadata

## Scenario

> ACME has elected to recency metadata to accompany their FOCUS data export. ACME provides a directory of recency metadata for each dataset they provide and each recency object is a single file.

Reviewer: ```suggestion
ACME has elected to add recency metadata to its FOCUS data export. ACME provides a directory of recency metadata for each dataset they provide and each recency object is a single file.
```

## Supplied Metadata

Metadata can be provided at a location such as `/FOCUS/metadata/recency/recency-1234-abcde-12345-abcde-12345.json`.

> The provided recency metadata for a time series dataset could look like this:

Reviewer: Are you intending to be prescriptive on the json or no? If yes, I'd expect us to say "should" vs. could, but if you're not intending to be prescriptive, could works and you can resolve this comment

Author: Until we get serious about metadata structure we have traditionally kept this as non prescriptive. If we change this to a should...I want to make it a must and do it everywhere in metadata. Until then we leave it as could.

```json
{

>   "Dataset": "1234-abcde-12345-abcde-12345",

Author: This is not sufficient, we need to have an identifier and preferably have a human readable name  as well.
